From: Michael Froman <mjfroman@mac.com>
Date: Fri, 29 Dec 2023 17:22:23 -0600
Subject: (tmp-cherry-pick) Revert "Set optimize_for_size for iOS perf tests."
 (573bb8552a)

This reverts commit f83da9dddddf0cfd421688cf68fa26bf3ecadd1d.

Reason for revert: Culprit has been found.

Original change's description:
> Set optimize_for_size for iOS perf tests.
>
> No-Try: True
> Bug: webrtc:15524
> Change-Id: I525f014f5c321000d520ef68a682f8ca1395bebd
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/323002
> Commit-Queue: Jeremy Leconte <jleconte@google.com>
> Reviewed-by: Jeremy Leconte <jleconte@google.com>
> Auto-Submit: Mirko Bonadei <mbonadei@webrtc.org>
> Cr-Commit-Position: refs/heads/main@{#40915}

Bug: webrtc:15524
Change-Id: I6785bf448f62d39a9eafe04bce6c778ba638c9ac
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/323660
Auto-Submit: Mirko Bonadei <mbonadei@webrtc.org>
Commit-Queue: Jeremy Leconte <jleconte@google.com>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Reviewed-by: Jeremy Leconte <jleconte@google.com>
Cr-Commit-Position: refs/heads/main@{#40943}
---
 tools_webrtc/mb/mb_config.pyl | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/tools_webrtc/mb/mb_config.pyl b/tools_webrtc/mb/mb_config.pyl
index ff37cd0812..6aea3c9cba 100644
--- a/tools_webrtc/mb/mb_config.pyl
+++ b/tools_webrtc/mb/mb_config.pyl
@@ -268,12 +268,10 @@
       'ios', 'debug_bot', 'arm64', 'ios_code_signing_identity_description',
       'xctest'
     ],
-    # TODO(bugs.webrtc.org/15524): Remove 'optimize_for_size' once the bug
-    # has been resolved.
     'ios_internal_pure_release_bot_arm64': [
       'ios', 'pure_release_bot', 'arm64',
       'ios_code_signing_identity_description', 'xctest',
-      'rtc_objc_test_prefix', 'optimize_for_size',
+      'rtc_objc_test_prefix',
     ],
     'ios_internal_release_bot_arm64': [
       'ios', 'release_bot', 'arm64', 'ios_code_signing_identity_description',
@@ -450,9 +448,6 @@
     'optimize_for_fuzzing': {
       'gn_args': 'optimize_for_fuzzing=true',
     },
-    'optimize_for_size': {
-      'gn_args': 'optimize_for_size=true',
-    },
     'partial_code_coverage_instrumentation': {
       'gn_args':
       'coverage_instrumentation_input_file="//.code-coverage/files_to_instrument.txt"'
